### PR TITLE
made it swipe better

### PR DESCRIPTION
**Pull request by ejohnson44**

### CODE REVIEW

**Comment by theJenix** [comment]
two enthusiastic :thumbsup:

**Comment by theJenix** [comment]
:thumbsup:

File: SoundStream/src/com/lastcrusade/soundstream/components/MusicLibraryFragment.java [hunk 1]
```diff
@@ -117,11 +114,34 @@ public void onDestroy() {
         unregisterReceivers();
     }
 
+    /* (non-Javadoc)
+     * @see com.lastcrusade.soundstream.components.MusicListFragment#onResume()
+     */
+    @Override
+    public void onResume() {
+        // TODO Auto-generated method stub
```
Comment (reidbaker) [inline SoundStream/src/com/lastcrusade/soundstream/components/MusicLibraryFragment.java]: remove

File: SoundStream/src/com/lastcrusade/soundstream/components/PlaylistFragment.java [hunk 2]
```diff
@@ -99,6 +100,8 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         super.onCreateView(inflater, container, savedInstanceState);
         View v = inflater.inflate(R.layout.list, container, false);
         setListAdapter(mPlayListAdapter);
+
```
Comment (reidbaker) [inline SoundStream/src/com/lastcrusade/soundstream/components/PlaylistFragment.java]: DO we need this white space

**Comment by ejohnson44** [comment]
Haha, it's not 100% yet, but it fixes some of the major problems you had with it and give you something that is easier to demo tomorrow. I will work on the smaller issues over the next week

**Pull request merged**